test/DynamoCoreTests/Updates/UpdatesTests.cs

```diff
@@ -47,5 +51,88 @@ public void UpdateManagerConfiguration_LoadTest()
             Assert.IsNull(UpdateManagerConfiguration.Load("filePath", updateManager));
             Assert.IsNull(UpdateManagerConfiguration.Load(Path.GetTempFileName(), updateManager));
         }
+
+        [Test]
+        [Category("UnitTests")]
+        public void UpdateManagerConfiguration_SaveTest()
+        {
+            var config = new UpdateManagerConfiguration();
+            Assert.DoesNotThrow(() => config.Save(Path.GetTempFileName(), updateManager));
+        }
+
+        [Test]
+        [Category("UnitTests")]
+        public void CheckNewerDailyBuildTest()
```

Comment:
Can we rename this test? This name is misleading as it seems it actually check newer daily builds

Reply:
Done, I consolidated the tests for the properties and it's now called UpdateManagerConfigurationPropertiesTest